src/components/ReportingConfig/EmailDeliveryMethodForm.jsx

```diff
@@ -29,7 +24,13 @@ const EmailDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
             })}
             data-hj-suppress
           />
-        </ValidationFormGroup>
+          <Form.Text>The email(s), one per line, where the report should be sent.</Form.Text>
+          {invalidFields.emailRaw && (
+          <Form.Control.Feedback type="invalid">
+            Required. One email per line. Emails must be formatted properly (email@domain.com)
+          </Form.Control.Feedback>
+          )}
+        </Form.Group>
         {config && (
           <div className="form-group">
```

Comment:
[suggestion] Might be a good idea to convert this custom `div.form-group` into a real `Form.Group`, `Form.Label`, and `Form.Checkbox`.